modules/nw-sriov-configuring-operator.adoc

```diff
@@ -5,9 +5,20 @@
 [id="configuring-sr-iov-operator_{context}"]
 = Configuring the SR-IOV Network Operator
 
+[IMPORTANT]
+====
+Modifying the SR-IOV Network Operator configuration is not normally necessary.
+The default configuration is recommended for most use cases.
+Complete the steps to modify the relevant configuration only if the default behavior of the Operator is not compatible with your use case.
+====
+
 The SR-IOV Network Operator adds the `SriovOperatorConfig.sriovnetwork.openshift.io` Custom Resource Definition (CRD) to {product-title}.
 You can configure the SR-IOV software compoenents by modifying the default 
-SriovOperatorConfig Custom Resource (CR).
+SriovOperatorConfig Custom Resource (CR). The Operator supports the following optional behavioral changes:
+
+- Disabling the Network Resources Injector
+- Disabling the Operator admission web controller
+- Configuring a custom node selector
```

Comment:
To make it more clear, we could say.
Configuring a custom node selector for the sriov-network-config-daemon. It runs on all the worker nodes by default.